tests/test_wave_01.py

```diff
@@ -119,12 +118,13 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    #raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
-
-@pytest.mark.skip()
+    assert updated_data["watched"][0]["title"] ==  MOVIE_TITLE_1
+    assert updated_data["watchlist"]== []
```

Comment:
This is essentially, although not exhaustively covered by the assertion on line 118. Expanding on what you wrote on line 125, you could test for the other key/values in that dictionary as well.